x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ml/dataframe/analyses/OutlierDetectionTests.java

```diff
@@ -44,12 +46,14 @@ public void testGetParams_GivenDefaults() {
     }
 
     public void testGetParams_GivenExplicitValues() {
-        OutlierDetection outlierDetection = new OutlierDetection(42, OutlierDetection.Method.LDOF);
+        OutlierDetection outlierDetection = new OutlierDetection(42, OutlierDetection.Method.LDOF, 0.42);
 
         Map<String, Object> params = outlierDetection.getParams();
 
-        assertThat(params.size(), equalTo(2));
+        assertThat(params.size(), equalTo(3));
         assertThat(params.get(OutlierDetection.N_NEIGHBORS.getPreferredName()), equalTo(42));
         assertThat(params.get(OutlierDetection.METHOD.getPreferredName()), equalTo(OutlierDetection.Method.LDOF));
+        assertThat((Double) params.get(OutlierDetection.MINIMUM_SCORE_TO_WRITE_FEATURE_INFLUENCE.getPreferredName()),
+            is(closeTo(0.42, 0.00001)));
```

Comment:
Alternatively, you could use scientific notation and decrease the threshold to "1e-9".